### PR TITLE
Mobile styling top sections

### PR DESCRIPTION
**Pull request by ivan-unfolds**
Mobile styling for -

* Banner Image.
* What We Do
* Our Work
* Get Involved
* Clients and Supporters

relates to #112

### CODE REVIEW

File: source/tachyons/_custom.css [hunk 1]
```diff
@@ -37,3 +42,11 @@ a {
 .transition-300 {
   transition: 0.3s ease
 }
+
+h1, h2, h3, h4, h5, h6 {
```
Comment (ivan-unfolds) [inline source/tachyons/_custom.css:46]: Not sure about these bottom two. They make everything look like the old site but I feel a little uneasy. What do you think? @andrewMacmurray

Comment (andrewMacmurray) [reply source/tachyons/_custom.css:46]: @ivanmauricio probably better to reference them as css vars, might be worth making a separate var for `roboto slab` with a couple of fallbacks (like Arial and Helvetica)

**Review by andrewMacmurray — COMMENTED** [review]
looks good, just one comment on the get involved section, the image sizes are still a little uneven

**Comment by ivan-unfolds** [comment]
@andrewMacmurray Hey think image sizes are good now, let me know what you think.

**Comment by andrewMacmurray** [comment]
@ivanmauricio beautiful 😻

**Pull request merged**